### PR TITLE
Updated tags.delete with bulk delete feature

### PR DESCRIPTION
**Pull request by tushar-balwani**
# Description

Updated tags.delete with bulk delete feature

- Deleting a single tag value:

```
>>> tio.tags.delete('00000000-0000-0000-0000-000000000000')
```

- Deleting multiple tag values:
```
>>> tio.tags.delete('00000000-0000-0000-0000-000000000000', '10000000-0000-0000-0000-000000000001')
```

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Added below test to check bulk delete tag values
 
- [x] Test `test_tags_delete_bulk_typeerror` to check any of value doesn't match to expected type
- [x] Test `test_tags_delete_bulk_unexpectedvalueerror` to check any of value doesn't match to expected value
- [x] Test `test_tags_delete_bulk_success` to check api is working

**Test Configuration**:
* Python Version(s) Tested: 3.8.6
* Tenable.sc version (if necessary):

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/io/test_tags.py [hunk 1]
```diff
@@ -82,292 +123,455 @@ def test_tags_create_category_success(api, tagcat):
...
         api.tags._tag_list_constructor([],
             api.tags._filterset_tags, None, 1)
 
 def test_tags_list_constructor_sort_unexpectedvalueerror(api):
+    '''
+    test to raise exception when sort param value does not match the choices.
+    '''
     with pytest.raises(UnexpectedValueError):
         api.tags._tag_list_constructor([],
             api.tags._filterset_tags, None, (('something_else'),))
```
Comment (mvalliammal) [inline tests/io/test_tags.py:401]: Access to a protected member of the _tags_list_constructor

Comment (tushar-balwani) [reply tests/io/test_tags.py]: Addressed above comment in next commit

Comment (tushar-balwani) [reply tests/io/test_tags.py]: Addressed above comment in next commit

**Review by valliammal — APPROVED** [review]
Approved the changes, looks good

**Review by levwais — APPROVED** [review]

**Pull request merged**